keyboards/keyhive/maypad/keymaps/keymap.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 codybender
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT_ortho_5x4(
+		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, 
+		KC_P7, KC_P8, KC_P9, KC_PPLS, 
+		KC_P4, KC_P5, KC_P6, KC_PPLS, 
+		KC_P1, KC_P2, KC_P3, KC_ENT, 
+		KC_P0, KC_P0, KC_PDOT, KC_ENT),
+
+	LAYOUT_ortho_5x4(
```

Comment:
This layer actually doesn't need to exist as it's completely transparent and there's no way to get to it.